tasks/coverage/snapshots/parser_typescript.snap

```diff
@@ -10204,6 +10201,13 @@ Expect to Parse: tasks/coverage/typescript/tests/cases/conformance/salsa/private
    ╰────
   help: Try insert a semicolon here
 
+  × Unexpected token
+   ╭─[typescript/tests/cases/compiler/parseInvalidNullableTypes.ts:1:30]
+ 1 │ function f1(a: string): a is ?string {
```

Comment:
Regarding the above comment, [TypeScript provides this error message](https://www.typescriptlang.org/play/?#code/GYVwdgxgLglg9mABMAjACgIYC5EGcoBOMYA5gJQ4aIy6ID8+RpiA3gFCKeIECmUIBJIRA82AXyA):

> '?' at the start of a type is not valid TypeScript syntax. Did you mean to write 'string | null | undefined'? (17020)